.buildkite/publish/publish-common.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [[ "${CURDIR:-}" == "" ]]; then
+  echo "!! CURDIR is not set. Exiting."
+  exit 2
+fi
+
+function realpath {
+  echo "$(cd "$(dirname "$1")" || exit; pwd)"/"$(basename "$1")";
+}
+
+export SCRIPT_DIR="$CURDIR"
+
+BUILDKITE_DIR=$(realpath "$(dirname "$SCRIPT_DIR")")
+PROJECT_ROOT=$(realpath "$(dirname "$BUILDKITE_DIR")")
+VERSION_PATH="$PROJECT_ROOT/product_version"
+VERSION=$(cat "$VERSION_PATH")
+#IS_SNAPSHOT=$(buildkite-agent meta-data get is_snapshot)
+
+export BUILDKITE_DIR
+export PROJECT_ROOT
+export VERSION
+
+if [[ "${IS_SNAPSHOT:-}" == "true" ]]; then
+  echo "Adding SNAPSHOT labeling"
+  export VERSION="${VERSION}-SNAPSHOT"
+fi
+
+export BASE_TAG_NAME="${DOCKER_IMAGE_NAME:-docker.elastic.co/integrations/crawler}"
+export DOCKERFILE_PATH="${DOCKERFILE_PATH:-Dockerfile}"
```

Comment:
@navarone-feekery now that we have a working Wolfi-based Dockerfile, do you think we could try using it here, at this point?

Reply:
I also think this PR should switch to wolfi, I'll work on that today

Reply:
It built fine with the wolfi image 🎉 
https://buildkite.com/elastic/crawler-docker-build-publish/builds/14